### PR TITLE
Fixes when has not target, in components has return null.

### PR DESCRIPTION
**Pull request by krailler**

### CODE REVIEW

File: src/Tween.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ const Tween = function (target, toData, props) {
   this.perFrame = Math.round(1000 / 60);
   // 注册，第一次进入执行注册
   this.register = false;
+  if (!this.target) return
```
Comment (jljsj33) [inline src/Tween.js]: 不可能为 null 的情况吧，，你是在什么情况下遇到这个问题？

**Comment by jljsj33** [comment]
or `this.target = target || {}`

**Comment by krailler** [comment]
I show you a better example ...

```
function RequireAuth({
    auth: {
        userLogged,
    },
    children,
}) {
    return userLogged ? children : null
}
```

When i use this component (Simple example)...
```
...
<RequireAuth><My Component /><RequireAuth/>
...
````
RequireAuth component will be render a none if user not logged.

**Comment by jljsj33** [comment]
thx, there is no problem in this situation.

Although the children is null, `tween-one` is aimed at the current tag, as long as component is not `""` or `RequireAuth`.

```jsx
...
<TweenOne component="div">
  <RequireAuth><My Component /><RequireAuth/>
</TweenOne>
...
```

or 

```jsx
function RequireAuth({
    auth: {
        userLogged,
    },
    children,
}) {
    return userLogged ? (<TweenOne>children</TweenOne>) : null
}
```


> If you want to add, please add this judgment to TweenOne.jsx. The current increase will cause errors in reStart and resetDefaultStyle. thx.

**Pull request closed without merging**